### PR TITLE
Rename helpers and common.py functions to lowercase

### PR DESCRIPTION
**Pull request by fernandog**
It will fix a lot of flake issues as those functions are used everywhere

### CODE REVIEW

File: medusa/search/core.py [hunk 1]
```diff
@@ -16,15 +16,17 @@
...
-
+"""Search core module."""
 import datetime
 import errno
 import os
 import threading
 import traceback
-from socket import timeout as SocketTimeout
+
+from socket import timeout as socket_timeout
```
Comment (p0psicles) [inline medusa/search/core.py]: Please double check this

File: medusa/search/core.py [hunk 2]
```diff
@@ -507,20 +509,20 @@ def searchProviders(show, episodes, forced_search=False, down_cur_quality=False,
...
 
             try:
-                searchResults = cur_provider.find_search_results(show, episodes, search_mode, forced_search,
-                                                                 down_cur_quality, manual_search, manual_search_type)
+                search_results = cur_provider.find_search_results(show, episodes, search_mode, forced_search,
+                                                                  down_cur_quality, manual_search, manual_search_type)
             except AuthException as e:
                 logger.log(u"Authentication error: " + ex(e), logger.ERROR)
                 break
-            except SocketTimeout as e:
```
Comment (p0psicles) [inline medusa/search/core.py]: Classes should be CamelCase.

File: medusa/search/proper.py [hunk 3]
```diff
@@ -26,26 +26,31 @@
 import threading
 import time
 import traceback
-from socket import timeout as SocketTimeout
+
+from socket import timeout as socket_timeout
```
Comment (p0psicles) [inline medusa/search/proper.py]: Was good as it was.

File: medusa/search/proper.py [hunk 4]
```diff
@@ -138,7 +141,7 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
                 logger.log('Authentication error: {error}'.format
                            (error=ex(e)), logger.DEBUG)
                 continue
-            except SocketTimeout as e:
+            except socket_timeout as e:
```
Comment (p0psicles) [inline medusa/search/proper.py]: Same

File: views/manage_massEdit.mako [hunk 5]
```diff
@@ -79,7 +79,6 @@
                                             initial_quality = int(quality_value)
                                         else:
                                             initial_quality = common.SD
-                                        allowed_qualities, preferred_qualities = common.Quality.splitQuality(initial_quality)
```
Comment (p0psicles) [inline views/manage_massEdit.mako]: Double check this pls.

File: views/partials/home/banner.mako [hunk 6]
```diff
@@ -46,18 +46,18 @@
     </tfoot>
     % if app.showQueueScheduler.action.loadingShowList:
         <tbody class="tablesorter-infoOnly">
-        % for curLoadingShow in app.showQueueScheduler.action.loadingShowList:
-            <% if curLoadingShow.show is not None and curLoadingShow.show in app.showList:
+        % for cur_loading_show in app.showQueueScheduler.action.loadingShowList:
```
Comment (p0psicles) [inline views/partials/home/banner.mako]: Didn't you change the actions to snake case?

Comment (fernandog) [inline views/partials/home/banner.mako]: Not scheduler actions

**Comment by fernandog** [comment]
@p0psicles any more change request?

**Pull request merged**